### DISCUSSION TITLE
No Sharing Button

### DISCUSSION THREAD

**Discussion by raniaamina** [photoprism/photoprism#3159] [Q&A] [answered]
Hi, it's my first time using PhotoPrisme
All works well, and no installation error that i face, but i can't find sharing button in context menu. Should i do something to show it?

I install via docker compose.

![gambar](https://user-images.githubusercontent.com/22146607/216125501-1748a9f4-887d-42e6-aaf6-198341d2946a.png)

Thanks in advance.

**Comment by graciousgrey** [comment, marked as answer]
At the moment is it only possible to share albums, not single files (that's planned). If you select an album and open the context menu the share button will be visible

   https://docs.photoprism.app/user-guide/share/

**Reply by raniaamina** [reply]
Ah, still notice it. Thanks for answer, i though it will same for single file and album.

Thanks!
